src/logger/NetHttpClientLogger_test.go

```diff
@@ -3,20 +3,63 @@ package logger
 import (
 	"fmt"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
-func TestNetHttpClientLogger(t *testing.T) {
-	testHelper := NewTestHelper()
+//Testing NET HTTP Logger
+
+func TestLogsGet(t *testing.T) {
+
+	netLogger := NewNetHttpClientLogger(nil)
+	helper := GetTestHelper()
+	resp, err := netLogger.Get(helper.demoURL)
+	fmt.Println(resp)
+	fmt.Println(err)
+	//Don't think we will need this
+	//assert.True(t, parsable(resp))
+	assert.Contains(t, resp, "[\"request_method\",\"GET\"]")
```

Comment:
So all the `assert.Contains` and  `assert.NotContains` unfortunately won't work since the `resp` is of type `net.http.Response` and not a `[][]string` like the message to be logged.

What you will need to do is set the `NetHttpClientLogger`'s `httpLogger` to a new `HttpLogger` with a queue.

So you will want to set it like this:
`queue := make([]string, 0)`
`netLogger.httpLogger = NewHttpLoggerQueue(queue)`
then 
`assert.Equal(t, true, strings.Contains(t, queue[0], "[\"request_method\",\"GET\"]")`

I'm actually stealing this from @kolin-newby's branch.

Reply:
was able to fix this for the first test function. I currently have errors for the function NewHttpLoggerQueue which makes sense since i don't have it pulled in yet. Let me know if this looks good and i will convert the rest.